lib/stack.dart

```diff
@@ -33,4 +33,16 @@ class Stack<T> {
 
   /// get the length of the stack.
   int get length => size();
+  
+  /// print stack
+  void print_() {
+    if (_list.isEmpty) {
```

Comment:
@AunSyedShah After this method runs if you run `isEmpty` it will be true because you used `removeLast` until the stack is empty

I suggest you use something like this
```
void print() {
  fo(T value in _list.reversed) {
    print(value);
  }
}
```

what do you think?

Reply:
I am looking for further logics and experiments regarding this. Till now the method I added seems to work fine. I would like to add that I am a newbie in open-source and learning, enjoying this too much.

here is my repo for implementation of the same above module for stack in cpp
[Stack Print in CPP](https://github.com/AunSyedShah/ds/blob/master/stack/printStack/printStack.cppl)

Reply:
The URL isn't working, is it a private repo? Please double check it

The method you've added works fine but it has side effects (makes the stack empty) and I believe that is not a suitable behavior for a print functionality

Reply:
Thank you so much for explaining. I got your point now. I'm going to fix it with pass by value method so stack will remain unaffected. Thanks.